app/models/force/object_base.rb

```diff
@@ -65,24 +66,48 @@ def add_salesforce_suffix(salesforce_fields)
 
     def float_to_currency(field_name, domain_fields)
       return unless @fields.domain[field_name]
+
       domain_fields[field_name] = ActionController::Base.helpers.number_to_currency(@fields.domain[field_name])
     end
 
     def currency_to_float(field_name, salesforce_fields)
       return nil unless @fields.salesforce[field_name]
+
       salesforce_fields[field_name] = salesforce_fields[field_name].gsub(/[$,]/, '').to_f
     end
 
+    # TODO: Automate this conversion for date objects.
```

Comment:
We can skip the automated date conversion in this PR and leave this TODO for a future PR